apps/theming/css/default.css

```
@@ -49,6 +49,7 @@
   --default-font-size: 15px;
   --animation-quick: 100ms;
   --animation-slow: 300ms;
+  --border-radius-element: 10px; /** Nextcloud 29+ | temp variable to use old design */
   --border-radius: 3px;
   --border-radius-large: 10px;
   --border-radius-rounded: 28px;
```

- Changed order to be consistent with other variables in the file and with `DefaultTheme.php`
- Changed comment to reflect new actual meaning

```suggestion
  --border-radius: 3px;
  --border-radius-large: 10px;
  --border-radius-rounded: 28px;
  /* Border radius of interactive elements such as buttons, input, navigation and list items. Available since Nextcloud 30. */
  --border-radius-element: 10px; 
```